src/identity/interaction/routing/AbsoluteInteractionRoute.ts

```diff
@@ -0,0 +1,16 @@
+import type { InteractionRoute } from './InteractionRoute';
+
+/**
+ * A route that returns the input string as path.
+ */
+export class AbsoluteInteractionRoute implements InteractionRoute {
+  private readonly path: string;
+
+  public constructor(path: string) {
+    this.path = path;
+  }
+
+  public getPath(): string {
```

Comment:
Why not just make `path` public then?

Reply:
Yea, I guess I should do that. My mind is still set very hard on "only expose functions" 😄.

Reply:
Discovered a good reason to do it. That would turn the `InteractionRoute` interface into `{ path: string }` which the Components.js generator is going to deconstruct. So for the `InteractionHandler` constructor
```ts
public constructor(route: InteractionRoute, source: InteractionHandler)
```
Components.js no longer expects a `"route"` entry, but a `"route_path"` entry.

Reply:
> That would turn the `InteractionRoute` interface into `{ path: string }`

Also if read-only (i.e., with getter)? Might be a bug then, @rubensworks?

Reply:
Adding `readonly` to the interface did not have an impact. But it does make sense I think, since we also use interfaces to define input parameters for constructors with a lot of variables and otherwise that would become unfeasible: https://github.com/solid/community-server/blob/d2870e5c8bc2d86b1a43db3a8e039f8b494657da/config/app/setup/handlers/setup.json#L17-L24

You could perhaps expect Components.js to handle this differently when the parameters are defined as `readonly` but that might cause confusion (and make @rubensworks's life more difficult 😉)

Reply:
Interfaces are indeed deconstructed atm. But we could definitely add something like a tsdoc option to disable this.